crates/bevy_asset/src/processor/mod.rs

```diff
@@ -1,3 +1,42 @@
+//! Asset processing in Bevy is a framework for automatically transforming artist-authored assets into the format that best suits the needs of your particular game.
+//!
+//! You can think of the asset processing system as a "build system" for assets.
+//! When an artist adds a new asset to the project, the asset processing system will automatically perform the specified processing steps on the asset.
+//! This can include things like converting a `.png` file into a compressed GPU-friendly texture format, compressing a `.wav` file to an `.ogg`, or generating mipmaps for a texture.
```

Comment:
Very very small nit I won't block on: I feel like the 3 examples are basically compressing twice, and then generating mipmaps for textures, which is very similar to the first. Maybe replace one of these with light baking or another example?